src/Symfony/Component/Mime/RawMessage.php

```diff
@@ -48,12 +46,19 @@ public function toIterable(): iterable
             return;
         }
 
-        $message = '';
-        foreach ($this->message as $chunk) {
-            $message .= $chunk;
-            yield $chunk;
+        if ($this->message instanceof \Generator) {
```

Comment:
If the message is a generator, we can only use it once, so we need to store the result.